app/main.py

```diff
@@ -1,3 +1,20 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable, Any
+from functools import wraps
+
+
+def cache(func: Callable) -> Callable:
+    dict__ = dict()
```

Comment:
That is okay, but in general `dict__ = {}`  syntax is more commonly used and is considered more Pythonic.
It is up to you whether to change it or not, btw.